host-controller/src/main/java/org/jboss/as/host/controller/DomainModelControllerService.java

```diff
@@ -897,7 +897,8 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
                     Notification notification = new Notification(ModelDescriptionConstants.BOOT_COMPLETE_NOTIFICATION, PathAddress.pathAddress(PathElement.pathElement(CORE_SERVICE, MANAGEMENT),
                             PathElement.pathElement(SERVICE, MANAGEMENT_OPERATIONS)), ControllerLogger.MGMT_OP_LOGGER.bootComplete());
                     getNotificationSupport().emit(notification);
-                    bootstrapListener.printBootStatistics();
+                    bootstrapListener.printBootStatistics(true, environment.getDomainConfigurationFile().getMainFile().getName()
+                            + ", " + environment.getHostConfigurationFile().getMainFile().getName());
```

Comment:
Wouldn't it be better to pass a variable argument and leave the format, in this case, separated by ", " to the logger itself?

I feel how the files will be printed is the responsibility of the logger message and not to the argument passed by to the printBootStatistics()